internal/pkg/ntp/ntp.go

```diff
@@ -181,17 +189,15 @@ func (syncer *Syncer) Run(ctx context.Context) {
 		case pollInterval == 0:
 			// first sync
 			pollInterval = syncer.MinPoll
-		case err != nil:
-			// error encountered, don't change the poll interval
 		case !spike && absDuration(resp.ClockOffset) > ExpectedAccuracy:
 			// huge offset, retry sync with minimum interval
 			pollInterval = syncer.MinPoll
-		case absDuration(resp.ClockOffset) < ExpectedAccuracy*100/25: // *0.25
```

Comment:
was this a bug, using `4` instead of `.0.25`?

Reply:
I believed so, and Andrey agreed it's weird. Comments said 0.25 and it was logical to have